src/webui/www/public/scripts/client.js

```diff
@@ -292,6 +293,7 @@ window.addEvent('load', function () {
                     var update_categories = false;
                     var full_update = (response['full_update'] === true);
                     if (full_update) {
+                        torrentsTableSelectedRows = torrentsTable.selectedRowsIds();
```

Comment:
What is the value of these IDs? The info-hash?

Reply:
Yeah exactly.